src/tests/routes/userSpec.js

```diff
@@ -1,6 +1,6 @@
 import chai from 'chai';
 import chaiHttp from 'chai-http';
-import app from '..';
+import app from '../..';
```

Comment:
why did you move the tests into the routes folder?

Reply:
The tests for routes are meant to be in the folder, I think that's why ttl had us set the test folder up like that